scripts/setup-ubuntu.sh

```diff
@@ -43,6 +43,7 @@ sudo --preserve-env apt update && sudo --preserve-env apt install -y libunwind-d
   ninja-build \
   checkinstall \
   git \
+  libboost-all-dev \
```

Comment:
Do we actually need that version of boost? It doesn't look like it, after all this version of boost is used and build and tests works. (Unless the aggregates failure is related to boost?)

Reply:
We will be changing boost 1.84.0 as the minimum required version soon.
This is needed for some Presto functions and we wanted to change the minimum in that PR.
It will be nice to have the build ready for that change.

Reply:
Ah I see, hm then we only want to build boost so we don't add another 15 minutes too each of the fuzzing steps. Maybe we can just use boost static, I'll look into it.

Reply:
Ah I see, hm then we only want to build boost so we don't add another 15 minutes too each of the fuzzing steps. Maybe we can just use boost static, I'll look into it.

Reply:
Yep, that works when actually building boost static in resolve_dependency :)